[1.21] Add most remaining schema documentation (#1567)
Shadows-of-Fire/Apotheosis: schema/affix/effect/OmneticAffix.md

# Description
OmneticAffix allows a tool to harvest blocks and have mining speed as if it were another tool type. For example, a sword with this affix could break blocks as if it were a pickaxe, axe, or shovel.

# Dependencies
This object references the following objects:
1. [Affix](../Affix.md)
2. [AffixDefinition](../AffixDefinition.md)

> 3. [LootRarity](../../loot/LootRarity.md)

Author: ```suggestion
1. [AffixDefinition](../AffixDefinition.md)
2. [LootRarity](../../loot/LootRarity.md)
```

# Schema
```js
{
    "type": "apotheosis:omnetic",        // [Mandatory] || The affix type identifier

>     "definition": AffixDefinition,        // [Mandatory] || The affix definition

Author: ```suggestion
    "definition": AffixDefinition,       // [Mandatory] || The affix definition
```

    "values": {                          // [Mandatory] || Per-rarity omnetic data
        LootRarity: {

>             "name": string,              // [Mandatory] || The name of the tool type (e.g., "pickaxe", "axe", "shovel")

Author: ```suggestion
            "name": string,              // [Mandatory] || The name of the tool class (e.g., "diamond" or "netherite")
```
